website/components/LandingPage/index.tsx

```diff
@@ -209,15 +209,15 @@ const DatasetCard = styled.li`
 const InReviewFlag = styled(FlexRowAlignCenter)`
   border-radius: 4px;
   padding: 1px 6px;
-  background-color: var(--color-statusflag-bg);
-  height: 22px;
+  border: solid 1px var(--color-statusflag-border);
```

Comment:
I thought the standard order for `border` was thickness then style, i.e. `border: 1px solid ...;`, but this also seems to work everywhere I try it? Optionally consider changing for consistency though, the other order is what's used elsewhere in the codebase:

```suggestion
  border: 1px solid var(--color-statusflag-border);
```